src/gh_pr_upsert/cli.py

```diff
@@ -1,13 +1,97 @@
+import sys
 from argparse import ArgumentParser
 from importlib.metadata import version
+from subprocess import CalledProcessError
 
+from gh_pr_upsert import core, git
+from gh_pr_upsert.exceptions import PRUpsertError
 
-def cli(_argv=None):  # pylint:disable=inconsistent-return-statements
-    parser = ArgumentParser()
+
+def cli(_argv=None):  # pylint:disable=too-complex
```

Comment:
Maybe there's some way to code design around pylint's warning here? But I doubt it's worth over-worrying about the code design of this little piece of automation tooling, and the code here isn't really complex there are just a lot of command line options and defaults to set.